packages/components/src/toolbar/stories/index.js

```diff
@@ -98,72 +122,99 @@ export const _default = () => {
 					{ icon: alignRight, title: 'Align right' },
 				] }
 			/>
+			<ToolbarDropdownMenu
+				icon={ chevronDown }
+				label="Select a direction"
+				controls={ [
+					{
+						title: 'Up',
+						icon: arrowUp,
+					},
+					{
+						title: 'Right',
+						icon: arrowRight,
+					},
+					{
+						title: 'Down',
+						icon: arrowDown,
+					},
+					{
+						title: 'Left',
+						icon: arrowLeft,
+					},
+				] }
+			/>
 		</Toolbar>
 	);
 };
 
-export const withoutGroup = () => {
+export const WithoutGroup = () => {
 	return (
 		// id is required for server side rendering
+		// eslint-disable-next-line no-restricted-syntax
 		<Toolbar label="Options" id="options-toolbar-without-group">
 			<ToolbarButton icon={ formatBold } label="Bold" isPressed />
 			<ToolbarButton icon={ formatItalic } label="Italic" />
 			<ToolbarButton icon={ link } label="Link" />
 		</Toolbar>
 	);
 };
-/* eslint-enable no-restricted-syntax */
 
-export const toolbars = () => {
+export const OnlyIcons = () => {
 	return (
-		<div>
-			<div style={ { padding: '20px' } }>
-				<h2>Icon-only Toolbar</h2>
-				<Toolbar>
-					<ToolbarButton icon={ formatBold } title="Bold" />
-					<ToolbarButton
-						icon={ formatItalic }
-						title="Italic"
-						isActive
-					/>
-					<ToolbarButton icon={ link } title="Link" />
-				</Toolbar>
-			</div>
+		// id is required for server side rendering
+		// eslint-disable-next-line no-restricted-syntax
+		<Toolbar>
```

Comment:
Copypasta?

Reply:
Yup! Not an issue anymore since I deleted all of those extra stories